CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Analyzer
 
+#### New features
+
+- The analyzer now infers the correct JSX quote style from `javascript.formatter.jsxQuoteStyle`, so code fixes will use the same quote style as the formatter. Contributed by @lucasweng
```

Comment:
That isn't true, though. In this PR you just fixed `useSortedClasses`, but there are many JSX-related rules that don't use this new feature yet. You're free to implement it in another PR if you want. As for now, we can remove this line